### PR TITLE
Make mouse pasting trigger search

### PR DESCRIPTION
**Pull request by Jaza**

### CODE REVIEW

**Comment by Jaza** [comment]
PR for fix provided by @demon101 in issue #882

**Comment by joallard** [comment]
Ref #882

File: src/selectize.js [hunk 1]
```diff
@@ -406,14 +406,15 @@ $.extend(Selectize.prototype, {
 		} else {
 			// If a regex or string is included, this will split the pasted
 			// input and create Items for each separate value
-			if (self.settings.splitOn) {
-				setTimeout(function() {
+			setTimeout(function() {
+				if (self.settings.splitOn) {
```
Comment (joallard) [inline src/selectize.js:410]: Know the rationale for this?

Comment (kerhong) [reply src/selectize.js:410]: I believe it's just to make `self.onKeyUp(e)` (like 461) always be triggered inside the delayed function. The issue is not with order of `setTimeout` and `if`, but in the fact that `self.onKeyUp(e)` also needs to be triggered after a delay.

Without `self.onKeyUp(e)` there is no code inside this function that actually handles the data.

**Comment by winzig** [comment]
@Jaza Why was this pull request closed? The issue still happens for me. Was a fix attempted in another pull request?

**Comment by winzig** [comment]
Still using this pasting fix to this day, and it's working. Would be awesome to get this merged.

**Pull request closed without merging**